design/workflow_examples.md

```diff
@@ -0,0 +1,78 @@
+# Workflow Examples
+
+- Detail a workflow for the deployment tool
+
+## Commands
+### endpoint
+The deployment tool communicates with a cluster fleet endpoint.
+This command sets which endpoint to communicate with. 
+Only one endpoint can be used at a time. 
+```
+$ formica endpoint http://127.0.0.1:4001
+$ formica endpoint
+http://127.0.0.1:4001
+```
+
+### ping
+For sanity's sake, let's add an easy way to test if we can actually reach the cluster.
+```
+$ formica ping && echo $?
+Cluster is reachable
+$ echo $?
+0
+$ formica ping --endpoint http://127.0.0.1:4002
+Cluster is not reachable
+$ echo $?
+1
+```
+
+### apply
+Configuration is available locally, and then applied to the cluster.
+The tool determines the delta between the state detailed locally and the cluster state, and then executes the changes required to set the cluster state to be equal to the local state.
+```
+$ ls
+[CONFIGURATION DETAILING HELLO_WORLD SERVICE]
```

Comment:
I want to keep the configuration format / details out of this pull request, so this is to show that there is _some_ configuration in the local directory, without specifying a format.